### ISSUE TITLE
Create modern team config branch

### ISSUE THREAD

**Issue by line47** [18F/federalist-uswds-template#10]
Create a configuration to be used for the modern team look.
https://federalist-modern-team-template.18f.gov/projects/

Preview: https://federalist-proxy.app.cloud.gov/preview/18f/federalist-uswds-template/modern-team-config/projects/

**Comment by line47** [comment]
I've added both _teams_ and _reports_ to the branch, @wslack and @davidewarren I'd love to get your thoughts on what I have in progress I welcome any and all comments 🎉 

Projects - https://federalist-proxy.app.cloud.gov/preview/18f/federalist-uswds-template/modern-team-config/projects/
Teams - https://federalist-proxy.app.cloud.gov/preview/18f/federalist-uswds-template/modern-team-config/team/

Here are links to each page type with the data that is being captured for each.
https://github.com/18F/federalist-uswds-template/blob/modern-team-config/_projects/project-1.md
https://github.com/18F/federalist-uswds-template/blob/modern-team-config/_team/team-member-01.md

I'm wondering about adding the report and team layouts to the uswds-jekyll theme so they can be used with other templates as well.

CC @hursey013 for your thoughts too

**Comment by wslack** [comment]
oooooooh. How would you add for some templates but not others?

**Comment by line47** [comment]
If we add the layouts at a higher level (uswds-jekyll) they would then be available for all of the federalist-template branches to use. Currently, as they are, they would just be available for this branch.

**Comment by wslack** [comment]
makes sense to me to add at a higher level then.

**Comment by line47** [comment]
Yeah, that's what I was thinking. I've started on that work here - https://github.com/18F/uswds-jekyll/issues/128

**Comment by line47** [comment]
@wslack and @davidewarren I'd ❤️ your feedback on this branch as well 👍

**Comment by adborden** [comment]
@wslack I think we should hold off on this until we validate the new Basic Report template first. I think if we focus validating the documentation around it, that will tell us a lot about how folks are using the templates and what the strategy should be with respect to the template recommendations.

**Comment by adborden** [comment]
Related issue https://github.com/18F/federalist/issues/1774
